### DISCUSSION TITLE
Mining reserve use

### DISCUSSION THREAD

**Discussion by lyjmry** [filecoin-project/FIPs#1075] [General]
This is the request of the majority of SPs, who have expressed their views on the cost of search support. They believe that they need additional storage to store unsealed and use high bandwidth to support retrieval, which will greatly increase their costs. Currently, except for some regions (Asia Pacific), it is difficult to maintain expenses. This can easily lead to distribution imbalance. If SPs reduce storage performance in order to reduce overhead, it will be detrimental to the entire network and prevent it from developing forward. When the SP's service capabilities are not very strong, it is difficult to attract commercial data for paid storage.
I think that the mining reserve of 300 million FIL will be delivered to SPs to improve their service capabilities and increase confidence in business data. The mining reserve of 300 million FIL was originally used to retrieve miners' rewards.

系广大SP的述求，他们在对检索支持方面的成本做出了看法。他们认为他们需要额外存储来存unsealed并且使用高带宽来支持检索，这将大大提高了他们的成本.目前除了一些地区以外（亚太）的其他地方，很难维持住开支。这样很容易导致分布失衡，如果SP们为了减少开销降低存储性能，对于整个网络是不利的，无法正向发展。当SP的服务能力不是很强时很难吸引商业数据进行付费存储。
我认为将3亿的挖矿储备开始交付给SP,以提高他们的服务能力，让商业数据对此增加信心，并且3亿FIL的挖矿储备金本来也是用于检索矿工的奖励的。

**Comment by willscott** [comment]
You maybe meant this as a comment on https://github.com/filecoin-project/FIPs/discussions/1030 rather than a separate discussion?

**Reply by lyjmry** [reply]
> You maybe meant this as a comment on #1030 rather than a separate discussion?

Hi, I'm glad you could let me see FIP1030. Maybe this has the same problem as mine, but they are discussing whether to destroy or not. We are discussing whether to enable it.
